Resources/translations/fr/content_type.fr.xlf

```diff
@@ -0,0 +1,147 @@
+      </trans-unit>
+      <trans-unit id="c3f683a33cfccf890c919f6432af7877" resname="content_type.edit">
+        <source>content_type.edit</source>
+        <target>Éditer</target>
+      </trans-unit>
+      <trans-unit id="972127730f863287531a351da529a802" resname="content_type.edit_title">
+        <source>content_type.edit_title</source>
+        <target>En cours d'édition</target>
+      </trans-unit>
+      <trans-unit id="bd31df6edfdfe616bf43312ee6ed4814" resname="content_type.content_count">
+        <source>content_type.content_count</source>
+        <target>{0} %count% élément de contenu|{1} %count% élément de contenu|]1,Inf[ %count% éléments de contenu</target>
+      </trans-unit>
+      <trans-unit id="978d4d6fcaabe95baf297bcddced946c" resname="content_type.modified_date">
+        <source>content_type.modified_date</source>
+        <target>Date de modification</target>
+      </trans-unit>
+      <trans-unit id="39f42e02559368a6b6ea2bb9f6de6724" resname="content_type.last_modified">
+        <source>content_type.last_modified</source>
+        <target>Dernière modification: %date% par %modifier%</target>
+      </trans-unit>
+      <trans-unit id="7c0ab0f708330c0d934c7e53a0cff93e" resname="content_type.description">
+        <source>content_type.description</source>
+        <target>Description</target>
+      </trans-unit>
+      <trans-unit id="e1d223825a9bf3ad0d1f810b7426bf3d" resname="content_type.name_schema">
+        <source>content_type.name_schema</source>
+        <target>Modèle de nom des contenus</target>
+      </trans-unit>
+      <trans-unit id="ad868db1b7d6a228c3b83176a33fd8fc" resname="content_type.url_alias_schema">
+        <source>content_type.url_alias_schema</source>
+        <target>Modèle de génération de l'URL</target>
+      </trans-unit>
+      <trans-unit id="b6a3aa854f989a0573db082c812c2d52" resname="content_type.container">
+        <source>content_type.container</source>
+        <target>Peut contenir des sous éléments</target>
+      </trans-unit>
+      <trans-unit id="975a8aa4deca92ef9cd5ffc42efd2467" resname="content_type.default_availability">
+        <source>content_type.default_availability</source>
+        <target>Disponible dans la langue par défaut même, si la traduction n'existe pas</target>
+      </trans-unit>
+      <trans-unit id="6361724b046699bacda179f0e7397de0" resname="content_type.always_available">
+        <source>content_type.always_available</source>
+        <target>Disponible</target>
+      </trans-unit>
+      <trans-unit id="5e276b6da192df3e9c622c398fc03d6b" resname="content_type.no_always_available">
+        <source>content_type.no_always_available</source>
+        <target>Non disponible</target>
+      </trans-unit>
+      <trans-unit id="9297b47a2cead7403adaab0f0ee6fdb2" resname="content_type.default_children_sorting">
+        <source>content_type.default_children_sorting</source>
+        <target>Tri par défaut des enfants</target>
```

Comment:
I don't know where this translation is used but I will suggest this translation
```Tri par défaut des sous-éléments```

(maybe a recollection of legacy ;) )